templates/demo/chains.json

```diff
@@ -1 +1,22 @@
-{}
```

Comment:
Why does the demo template now have a pre-filled chains.json?

Reply:
This shouldn't be in here. Not sure how this slipped through, but I'll remove this change from this commit.